helper/pluginutils/hclutils/util.go

```diff
@@ -72,3 +75,15 @@ func GetStdlibFuncs() map[string]function.Function {
 		"upper":      stdlib.UpperFunc,
 	}
 }
+
+func formattedDiagnosticErrors(diag hcl.Diagnostics) []error {
+	var errs []error
+	for _, d := range diag {
+		if d.Summary == "Extraneous JSON object property" {
```

Comment:
I would add a comment pointing to where this text originates, so future us and notice when it changes or if this is no longer needed.

Reply:
:+1: Added the comment before the method